setup.py

```diff
@@ -1,7 +1,14 @@
 #!/usr/bin/env python
 
+import os
+import sys
 from setuptools import setup, find_packages
 
+# Import directly from version.py by adding to sys.path
+version_path = os.path.join(os.path.dirname(__file__), 'pyhf')
+sys.path.append(version_path)
+from version import __version__
+
```

Comment:
do we need this? shouldn't bump version just ensure for us that the various files are set to the same value?

Reply:
This shouldn't be needed. You should be able to do `from pyhf.version import __version__`

Reply:
I think this is needed, as in a clean install it doesn't know about `pyhf` yet. [This can be seen in the Travis builds with errors like this one](https://travis-ci.org/diana-hep/pyhf/jobs/439806439#L488-L489):

```
$ pip install --ignore-installed -U -q -e .[complete]
Command "python setup.py egg_info" failed with error code 1 in /home/travis/build/diana-hep/pyhf/
```

Reply:
Python, by default, will treat any folder with `__init__.py` as a module, even if it has not been installed. So I'm not sure why that won't work.

Reply:
> shouldn't bumpversion just ensure for us that the various files are set to the same value?

Oh yeah. This would be better.